src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -48,16 +53,17 @@ export class List extends Setup {
         PerformanceMarkers.FETCH_DEVICES_MARKER_KEY
     )!;
 
-    public async run(direct: boolean = false): Promise<any> {
-        if (direct) {
-            await this.init(); // ensure init first
-        }
+    public async run(): Promise<any> {
```

Comment:
The `List` command should not change for now and should not run setup requirements at all. We use this command in our VSCode extension and we don't have a way there to tell the command to skip its requirement checks, which is why we decided to have this command to not do checks at all. You should revert the changes to this file (and its unit tests) for now.

Reply:
I didn't do a full revert but removed running requirement processor and updated the test accordingly.